src/components/Sigmet/SigmetEditMode.jsx

```diff
@@ -75,21 +74,42 @@ class SigmetEditMode extends PureComponent {
     }
   }
 
-  verifySigmetDebounced (sigmetObject) {
+  verifySigmet (sigmetObject) {
     const { dispatch, actions } = this.props;
     dispatch(actions.verifySigmetAction(sigmetObject));
   }
 
-  componentWillReceiveProps (nextProps) {
-    if (nextProps.sigmet && this.props.sigmet && nextProps.geojson && this.props.geojson) {
-      if (nextProps.sigmet !== this.props.sigmet || nextProps.geojson !== this.props.geojson) {
-        this.verifySigmetDebounced(nextProps.sigmet);
+  componentDidUpdate (prevProps) {
+    if (prevProps.sigmet && this.props.sigmet && prevProps.geojson && this.props.geojson) {
+      /* The issue with the looping updates of the TAC message in the sigmet, is because the tac property is inside the sigmet object.
+        When you update the tac, the sigmet is different triggering a new update.
+        Another issue is that when the verifySigmet function is bound with a debounce function, updates occur 250 ms later, 
+        causing problems with synchronization of objects. Objects are updated with the wrong (old) data.
+
+        TODO DECIDE: Keep it like this or refactor the tac property outside the sigmet object, just like the geojson. For now we do a deep compare without the TAC property.
+      */
+      let sigmetIsDifferent = false;
+      if (prevProps.geojson !== this.props.geojson) {
+        sigmetIsDifferent = true;
+      } else {
+        Object.keys(prevProps.sigmet).forEach((sigmetProperty) => {
```

Comment:
- Should we really compare each prop?
- Different instances of data types with the same content, might not be marked as equal